frontend/src/app/accountCreation/MfaSmsVerify/MfaSmsVerify.tsx

```diff
@@ -1,15 +1,16 @@
 import { MfaVerify } from "../MfaVerify/MfaVerify";
 
 interface Props {
-  phoneNumber: string;
+  location: { state: { phoneNumber: string } };
 }
 
 export const MfaSmsVerify = (props: Props) => (
   <MfaVerify
     hint={
       <>
-        We’ve sent a text message (SMS) to <b>{props.phoneNumber}</b>. It will
-        expire in 10 minutes.
+        We’ve sent a text message (SMS) to{" "}
+        <b>{props.location?.state?.phoneNumber || ""}</b>. It will expire in 10
+        minutes.
```

Comment:
Do we need to give the user a button to re-request the activation code for all of these verification pages? The endpoint for that is `/resend-activation-passcode`. (Note that it's not at all written on the backend yet, lol, so if you want to wait to add it to these verification pages that's totally fine. It might just be a good idea to include a TODO)

Reply:
I ended up implementing the service function and attaching it to the "Send another code" button in `MfaVerify` :+1: